src/DevBetterWeb.Web/wwwroot/js/videos.js

```diff
@@ -1,5 +1,48 @@
 ﻿$(document).ready(function () {
-
-   
-
+    var userRole = '@(User.IsInRole("Administrators") ? "true" : "false")';
```

Comment:
This code isn't going to execute on the server since this is just a static js file.

Reply:
I think it will work as I called it like this 
```csharp
@section scripts{
    <script src="~/js/videos.js" asp-append-version="true"></script>
}
```

Reply:
Check in the browser in dev tools.

And even if it does work, what prevents any end user from changing false to true themselves in client code?

Reply:
it is working and the user can change the role in client code but the page will not load as it is protected in the backend also 
it is using to hide/show by role in client code only.